sql-plan-management.md

```diff
@@ -287,3 +287,40 @@ Because the baseline evolution automatically creates a new binding, when the que
     | `max_execution_time` | The longest duration for a query. |
 
 + `read_from_storage` is a special hint in that it specifies whether to read data from TiKV or from TiFlash when reading tables. Because TiDB provides isolation reads, when the isolation condition changes, this hint has a great influence on the evolved execution plan. Therefore, when this hint exists in the initially created binding, TiDB ignores all its evolved bindings.
+
+## Upgrade Checklist
+
+SQL Plan Management (SPM) may fail in upgrade due to compatibility issues. You need to check the environment before upgrade to make sure it succeeds. 
+
+* When you upgrade from a version earlier than v5.2.0 (that is, v4.0, v5.0, and v5.1) to the current version, make sure `tidb_evolve_plan_baselines` is disabled. If not, you must disable it before upgrade. Perform the following steps. 
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    -- Check whether `tidb_evolve_plan_baselines` is disabled in the earlier version. 
+  
+    select @@global.tidb_evolve_plan_baselines;
+  
+    -- If `tidb_evolve_plan_baselines` is still enabled, disable it. 
+  
+    set global tidb_evolve_plan_baselines = off;
+    ```
+
+* Before you upgrade from v4.0 to the current version, you must check whether the grammar of all the SQL statements corresponding to available SQL bindings is correct in the new version. If any grammatical errors, delete the corresponding SQL bindings. Perform the following steps.
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    -- Check the SQL statements corresponding to available SQL bindings in the version to be upgraded.
+  
+    select bind_sql from mysql.bind_info where source != 'builtin' and status = 'using';
```

Comment:
```suggestion
    select bind_sql from mysql.bind_info where status = 'using';
```
I'll change the Chinese doc at the same time. It's a little mistake. It does not affect correctness, it just makes the SQL simpler.